Makefile.in

```diff
@@ -50,3 +53,9 @@ install: all
 
 distcheck:
 	@$(ECHO) @PACKAGE_VERSION@
+
+dist:
+	autoconf
```

Comment:
This misses calls to `autoheader` and `aclocal`.
Option A: Call `autoreconf -f -i`.
Option B: Add
```make
config.h.in: configure.in
    $(AUTOHEADER)

configure: configure.in config.h.in
    $(ACLOCAL)
    $(AUTOCONF)

dist: configure config.h.in
(...)
```

Reply:
I was not aware that we can use autoheader. So I guess we should remove config.h.in as well.

I do not have an aclocal command installed, AFAIK  we do not need it.